Configuration/TCA/tx_securefilemount_folder.php

```diff
@@ -9,10 +9,9 @@
         'iconfile' => 'EXT:secure_filemount/Resources/Public/Icons/Extension.svg',
         'tstamp' => 'tstamp',
         'crdate' => 'crdate',
-        'cruser_id' => 'cruser_id',
```

Comment:
Really remove it (for v11) ? In v12 this is stripped from core and handled over the history, but still it would make sense for v11. What's your decision on this on ?

Reply:
override in TCA/Overrides for Version compat in v11 done